packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -0,0 +1,547 @@
+---
+title: Status
+description: Use a status pattern to inform a user of the dynamic state or condition of an object.
```

Comment:
We recently reworked the pattern descriptions to be a little more aligned in their formatting, so it might make sense to word this one like "The status pattern informs the user of the dynamic state or condition of an object." Here are the others: https://docs.google.com/document/d/1T6nckBmrkIuPTMKz1H1PXqU4nm-MgigUIOa1tz8q9_M/edit#